docs/openapi/components/schemas/credentials/BillOfLadingCredential.yml

```diff
@@ -354,9 +354,9 @@ example: |-
     },
     "proof": {
       "type": "Ed25519Signature2018",
-      "created": "2022-11-29T07:37:53Z",
+      "created": "2022-12-20T15:30:19Z",
```

Comment:
@ipbyrne, you are re-signing all credentials for no reason. You shouldn't be touching the credentials which aren't actually part of the PR's scope. It makes it hard to understand the actual impact of the PR and causes unnecessary merge conflicts.
Please revert the re-signing of credentials which aren't needed. 
Next time, just run the `regenerate` script, it takes care of all this.

Reply:
Beyond the unnecessary resigns, the PR is great